media/transcoder/job_test.py

```diff
@@ -51,14 +52,17 @@ def test_job_operations(capsys):
     # Enable the following API on the test project:
     # *   Transcoder API
 
-    _clean_bucket()
+    _delete_bucket()
```

Comment:
Could you tweak these tests to use pytest fixtures instead? This increases the likelihood resources are cleaned up even if the test happens to fail midway.

* https://github.com/GoogleCloudPlatform/python-docs-samples/blob/a777e9d760bcf6b678474f930d9e9a329ddcd30f/storage/cloud-client/snippets_test.py#L96-L105
* https://github.com/GoogleCloudPlatform/python-docs-samples/blob/a777e9d760bcf6b678474f930d9e9a329ddcd30f/storage/cloud-client/snippets_test.py#L126-L132

Reply:
Done